src/jvm/main/org/jetbrains/kotlinx/lincheck/transformation/LincheckClassVisitor.kt

```diff
@@ -121,13 +122,159 @@ internal class LincheckClassVisitor(
             sv.analyzer = aa
             aa
         }
+        mv = CoverageBytecodeFilter(
```

Comment:
Please add a comment warning that `CoverageBytecodeFilter` should be placed after `SharedMemoryAccessTransformer` 

(Perhaps the relative order to some other transformers is also important? If so, please also mention this.)